src/common/DirectoryTreeView.tsx

```diff
@@ -0,0 +1,79 @@
+import TreeView from '@mui/lab/TreeView'
```

Comment:
I think we need to be careful about which components end up in `/common`. This is a component designed for a specific purpose. It is not a generic, reusable component and therefore shouldn't be in this directory imo.